packages/icon/package.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os, subprocess
+from spack import *
+
+
+class Icon(AutotoolsPackage):
+    """The ICON modelling framework is a joint project between the
+    German Weather Service and the
+    Max Planck Institute for Meteorology for
+    developing a unified next-generation global numerical weather prediction and
+    climate modelling system. The ICON model has been introduced into DWD's
+    operational forecast system in January 2015."""
+
+    homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
+    url = 'https://gitlab.dkrz.de/icon/icon-cscs/-/archive/mc10_for_icon-2.6.x-rc/icon-cscs-mc10_for_icon-2.6.x-rc.tar.gz'
```

Comment:
This is the latest official release: https://gitlab.dkrz.de/icon/icon/-/archive/icon-2.6.2.2/icon-icon-2.6.2.2.tar.gz
The version you are pointing at is the branch for the QUBICC development and not even everything will get back to the icon main repository.

Reply:
Ok, thx!